ERCS/erc-7786.md

```diff
@@ -109,6 +116,14 @@ After a sender has invoked `sendMessage`, further action MAY be required by the
 
 The exact interface for any such action is out of scope of this ERC. If the `postProcessingOwner` attribute is supported and present, such actions MUST be restricted to the specified account, otherwise they MUST be able to be performed by any party in a way that MUST NOT be able to compromise the eventual receipt of the message.
 
+#### `requestRelay`
+
+Requests that the message identified by `outboxId` be delivered to the receiver (i.e., relayed).
+
+Pre-pays via `msg.value` for the execution of the message on the receiver up to `gasLimit`. A `relayerFee` should be deducted from this to compensate the relayer.
+
+Any surplus after execution at destination MUST be refunded to `refundReceiver` on the destination chain.
+
```

Comment:
We probably want a blurb about how relaying should be permissionless. Something like:

Relaying should be permissionless. Callers should be free to self relay or use a different relayer service, in which case they should not need to call `requestRelay` on the gateway. Messages should be considered valid and ready for execution without `requestRelay` being called. Relaying can be a competitive marketplace, and no relayer should be relied on for liveness of the protocol.

Reply:
I went with the simpler version below because I think most of the above becomes tricky with L2s.

> Note that relaying SHOULD be permissionless. No relayer should be relied on for liveness of the protocol.